src/EventGridEmulator/Dockerfile

```diff
@@ -1,18 +1,18 @@
-﻿FROM mcr.microsoft.com/dotnet/aspnet:7.0-alpine AS base
+﻿FROM mcr.microsoft.com/dotnet/aspnet:8.0-alpine AS base
 WORKDIR /app
 EXPOSE 6500
 ENV DOTNET_CLI_TELEMETRY_OPTOUT=true
 ENV DOTNET_ENVIRONMENT=Production
 ENV ASPNETCORE_URLS=http://+:6500
 
-FROM mcr.microsoft.com/dotnet/sdk:7.0-alpine AS publish
+FROM mcr.microsoft.com/dotnet/sdk:8.0-alpine AS publish
 WORKDIR /src
 COPY . .
 WORKDIR "/src/EventGridEmulator"
 RUN dotnet publish "EventGridEmulator.csproj" -c Release -r linux-musl-x64 --no-self-contained --nologo -p:RunAnalyzers=false -o /app/publish
 
 FROM base AS final
-RUN adduser --disabled-password --home /app --gecos '' app && chown -R app /app
```

Comment:
in the dotnet 8 image sdk, the user app is already present by default and that duplicate creation was causing an error: https://devblogs.microsoft.com/dotnet/securing-containers-with-rootless/